package.json

```diff
@@ -74,7 +74,7 @@
     "broccoli-babel-transpiler": "^5.6.1",
     "broccoli-concat": "^3.0.1",
     "broccoli-debug": "^0.6.0",
-    "broccoli-dependency-funnel": "^1.0.2",
+    "broccoli-dependency-funnel": "2.0.0",
```

Comment:
Also, can we use `^` instead of exact match?  Just in case we need to fix any bugs or something...